root/area/AreaArtists.js

```diff
@@ -18,7 +18,7 @@ import AreaLayout from './AreaLayout';
 type Props = {
   +$c: CatalystContextT,
   +area: AreaT,
-  +artists: $ReadOnlyArray<ArtistT>,
+  +artists: ?$ReadOnlyArray<ArtistT>,
```

Comment:
The code was doing `artists && artists.length > 0` below, so either the type was wrong here or the check was unnecessary. I changed the type in this case and elsewhere, though in other cases I just removed the additional check if I was fairly sure it couldn't be null/undefined.